float64[1] x
float64 t
float64[1] setpoint
